packages/stress/src/utils/storage.ts

```diff
@@ -1,7 +1,7 @@
-import { dlogger } from '@river-build/dlog'
 import Redis from 'ioredis'
+import { getLogger } from './logger'
 
-const logger = dlogger('stress:storage')
+const logger = getLogger('storage')
```

Comment:
@miguel-nascimento we may want to keep the `stress:x` namespacing. this was an artifact from dlog which internally depends on the `debug` npm library, which allowed us to filter logs not just based on log levels but also log namespaces. it's not urgent to implement this functionality for stress tests with pino, but we should probably at least leave the namespaces there. see screenshot for references to this namespace:

<img width="536" alt="image" src="https://github.com/user-attachments/assets/37e10362-be30-40c4-9dae-a63b582393fa">